indigo_app/views/base.py

```diff
@@ -39,6 +41,9 @@ class AbstractAuthedIndigoView(PermissionRequiredMixin, IndigoJSViewMixin):
     must_accept_terms = True
 
     def dispatch(self, request, *args, **kwargs):
+        if is_maintenance_mode(request):
```

Comment:
This prevents all views that require authentication from loading.

Reply:
nice